### PR TITLE
Add JSON payload transformation examples for Hiveeyes and Bee Observer (TTN, Kotori, BEEP)

### PR DESCRIPTION
**Pull request by amotl**
Hi there,

this is what people from the [Hiveeyes](https://hiveeyes.org/) community conceived the other day in order to converge data payloads from [The Things Network (TTN)](https://de.wikipedia.org/wiki/The_Things_Network) towards [Kotori](https://getkotori.org/) and [BEEP](https://beep.nl/). Thanks @thiasB and @ClemensGruber.

See also:
- https://community.hiveeyes.org/t/ttn-daten-an-kotori-weiterleiten/1422/5
- https://community.hiveeyes.org/t/datenweiterleitung-via-ttn-lora-zu-hiveeyes-bob-und-beep-einrichten/3197
- https://github.com/hiveeyes/terkin-datalogger/tree/0.10.0/client/TTN

What we are doing here is an attempt to conceive an execution engine as an open source alternative to PutsReq, which sadly went closed source the other day - see [1] and [2].

Thanks a stack to @elbart for making this possible!

With kind regards,
Andreas.

[1] https://www.michaelfrew.com/putsreq/
[2] https://github.com/EvermeshSolutions/putsreq

### CODE REVIEW

**Comment by amotl** [comment]
@ClemensGruber asked the other day: "[Is it possible to call `request.forwardTo` twice?](https://github.com/EvermeshSolutions/putsreq/issues/71)".

@elbart just told me about one hour ago:
> The machinery is now capable of making actual outbound requests. The implementation is still incomplete, but it's a start. Also, multiple outbound HTTP requests within a single recipe already work.

File: tests/test_examples_applications_ttn.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    print(stdout)
+
+    # Check STDOUT
+    data = json.loads(stdout)
+    print(data)
+    assert data["method"] == "POST"
+
+    with open(basedir / "ttn-to-hiveeyes-egress.json", "r") as f:
+        outcome = json.load(f)
+    assert json.loads(data["body"]) == outcome
```
Comment (elbart) [inline tests/test_examples_applications_ttn.py]: I think it's cleaner to put the `assert` within the Context Manager

File: tests/test_examples_applications_ttn.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    print(stdout)
+
+    # Check STDOUT
+    data = json.loads(stdout)
+    print(data)
+    assert data["method"] == "POST"
+
+    with open(basedir / "ttn-to-beeobserver-egress.json", "r") as f:
+        outcome = json.load(f)
+    assert json.loads(data["body"]) == outcome
```
Comment (elbart) [inline tests/test_examples_applications_ttn.py]: same as above

**Review by elbart — APPROVED** [review]
I am super impressed, that this works already! Thanks a lot ;)

**Pull request merged**